packages/lit-dev-content/site/docs/v3/components/decorators.md

```diff
@@ -23,19 +34,14 @@ Stage 3 means that the specification text is complete, and ready for browsers to
 
 {% endaside %}
 
+Previous versions of the decorators proposal are also supported by some compilers, most notably TypeScript's "experimental decorators" which Lit has supported since its inception.
 
+Lit decorators work as both TypeScript experimental decorators and standard decorators. We currently recommend that TypeScript users use experimental decorator mode because the JavaScript output is much smaller than in standard decorator mode.
 
-Lit supplies a set of decorators that reduce the amount of boilerplate code you need to write when defining a component. For example, the `@customElement` and `@property` decorators make a basic element definition more compact:
+This means that TypeScript-based Lit projects should have `"experimentalDecorators": true` in their tsconfig.
+
+See the [Enabling decorators](#enabling-decorators) section for more information.
```

Comment:
If we move the `{#custom-element}` description up, and move the `## Standard decorators` section to the bottom, this line can be omitted.